src/hazardous/kdf/scrypt.rs

```diff
@@ -0,0 +1,445 @@
+// MIT License
+
+// Copyright (c) 2020-2025 The orion Developers
```

Comment:
```suggestion
// Copyright (c) 2025 The orion Developers
```
Copyright year starts when code is added.